workshop/content/exercises/create-pipeline.adoc

```diff
@@ -72,11 +72,16 @@ image:../images/console-import-yaml-1.png[OpenShift Console - Import Yaml Menu]
 
 image:../images/console-import-yaml-2.png[OpenShift Console - Import Yaml]
 
-You can check the list of `pipelines` you have created using the [Tekton CLI](https://github.com/tektoncd/cli/releases)(`tkn`):
```

Comment:
Why removing the link. As documentation, having this directing to the repo where the tool is produced can be handy.

Reply:
My thought was that this would be distracting for those doing the workshop. Is it likely that someone doing the workshop will stop to click on this link and read more? 

There's a link to the CLI under additional resources, and I am sure people will be capable of finding it after the workshop if they want more information.